### PR TITLE
Download won't add "playbook-" prefix to playbooks

### PR DESCRIPTION
**Pull request by ohaim1008**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/24581

## Description
When using the download command it's adding "playbook-" prefix.
Not needed anymore - so removed.

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: demisto_sdk/commands/download/tests/downloader_test.py [hunk 1]
```diff
@@ -644,10 +644,14 @@ def test_merge_new_file(self, tmp_path):
             output_dir_path = f'{temp_dir}/{entity}'
             os.mkdir(output_dir_path)
             old_file_path = param['custom_content_object']['path']
-            new_file_path = f'{output_dir_path}/{os.path.basename(old_file_path)}'
+            base_name = os.path.basename(old_file_path)
+            if base_name.startswith("playbook-"):
+                base_name = base_name[(len("playbook-")):]
+            new_file_path = f'{output_dir_path}/{base_name}'
```
Comment (bakatzir) [inline demisto_sdk/commands/download/tests/downloader_test.py:650]: if you handled it in the code, you want to test that it does not contain `playbook-` prefix. so why do you remove it here?

Comment (ohaim1008) [reply demisto_sdk/commands/download/tests/downloader_test.py:650]: right, removed it

**Review by guyfreund — CHANGES_REQUESTED** [review]
Overall looking good, but pay attention that you should handle both cases, existing file & new file.

What I suggest, edit the `update_file_prefix` function, it will handle both cases.

If you have any questions regarding how is better to implement this fix, feel free to ask.

File: demisto_sdk/commands/download/tests/downloader_test.py [hunk 2]
```diff
@@ -61,8 +61,8 @@ def setup(self):
         tests_path = self.tmp_path / 'tests'
         tests_env_path = tests_path / 'tests_env'
         tests_data_path = tests_path / 'tests_data'
-        shutil.copytree(src='demisto_sdk/commands/download/tests/tests_env', dst=str(tests_env_path))
-        shutil.copytree(src='demisto_sdk/commands/download/tests/tests_data', dst=str(tests_data_path))
+        shutil.copytree(src='/Users/ohaim/dev/demisto/demisto-sdk/demisto_sdk/commands/download/tests/tests_data', dst=str(tests_env_path))
```
Comment (guyfreund) [inline demisto_sdk/commands/download/tests/downloader_test.py:64]: You can't use your local paths, the paths should be within the demisto-sdk repo.
```suggestion
        shutil.copytree(src='demisto_sdk/commands/download/tests/tests_data', dst=str(tests_env_path))
```

File: demisto_sdk/commands/download/tests/downloader_test.py [hunk 3]
```diff
@@ -61,8 +61,8 @@ def setup(self):
...
+        shutil.copytree(src='/Users/ohaim/dev/demisto/demisto-sdk/demisto_sdk/commands/download/tests/tests_data', dst=str(tests_data_path))
```
Comment (guyfreund) [inline demisto_sdk/commands/download/tests/downloader_test.py:65]: You can't use your local paths, the paths should be within the demisto-sdk repo.
```suggestion
        shutil.copytree(src='demisto_sdk/commands/download/tests/tests_data', dst=str(tests_data_path))
```

**Review by guyfreund — CHANGES_REQUESTED** [review]

File: demisto_sdk/commands/download/tests/downloader_test.py (hunk 2 above)
Comment (ohaim1008) [reply demisto_sdk/commands/download/tests/downloader_test.py:64]: used it for local checks, forgot to remove

**Review by bakatzir — COMMENTED** [review]
Please add a line to the changelog

File: demisto_sdk/commands/download/downloader.py [hunk 4]
```diff
@@ -308,6 +308,8 @@ def update_file_prefix(file_name: str) -> str:
         """
         Custom content scripts are prefixed with automation instead of script.
         """
```
Comment (guyfreund) [inline demisto_sdk/commands/download/downloader.py:310]: Update description please

**Review by guyfreund — CHANGES_REQUESTED** [review]

**Review by bakatzir — APPROVED** [review]

**Review by guyfreund — APPROVED** [review]

**Pull request merged**